bank/services/stoneBaas/contracts.go

```diff
@@ -61,5 +61,17 @@ const templateRequest = `
 			{{end}}
 		}
 	{{end}}
+	{{if .Title.HasDiscounts}}
+		{{$length := len .Title.Discounts}}
+		,"discounts": [
+			{{range $index, $element := .Title.Discounts}}
+				{
+					"date": "{{.Date}}",
+					"exponent": -2,
+					"value": {{getValue (float64ToString "%.2f" .Percentage)}}
+				}{{getDiscountSeparator $index $length}}
```

Comment:
Opa, talvez fique mais simples a leitura passando assim, como o index começa com 0, do que ficando dentro do método getDiscountSeparator a verificação 😀 
```suggestion
				}{{getDiscountSeparator ($index+1) $length}}
```

Reply:
Infelizmente, não é possível fazer operações aritméticas no template nativo do go, apenas nas funções em que ele chama :c